README.md

```diff
@@ -6,6 +6,11 @@ UDH is a Commander format where your Commander can be any rarity, but the rest o
 
 This [Scryfall search query](https://scryfall.com/search?q=f%3Aedh+r%3C%3Du+-%28%22Cranial+Plating%22+or+%22Demonic+Tutor%22+or+%22Empty+the+Warrens%22+or+%22Frantic+Search%22+or+%22Skullclamp%22+or+%22Tendrils+of+Agony%22%29) contains all the legal cards in the format.
 
+Rarities used from the following sets are excluded:
+- [Renaissance](https://scryfall.com/sets/ren?as=grid&order=name)
+- [Tempest Remastered](https://scryfall.com/sets/tpr?as=grid&order=name)
```

Comment:
I don't understand why Tempest Remastered is getting banned here.

It's an English set, and is fundamentally identical as the other online-only [Masters sets](https://scryfall.com/sets?type=masters&lang=en)
Either we don't ban `TPR`, or we do ban `MED`, `ME2`, `ME3`, `ME4`, and `VMA`.
The current ruling is very inconsistent.

Reply:
It was something that was talked about at the beginning of the UDH shenanigans, I was simply following those discussions more than anything. 
I believe the argument was there had to be a paper printing of the card at the rarity and VMA/ME sets didn't downshift anything? I may be wrong on the mark with that but if that is the case, please let me know.

Reply:
[Plenty of stuff](https://scryfall.com/search?q=(e%3Ame1%20or%20e%3Ame2%20or%20e%3Ame3%20or%20e%3Ame4%20or%20e%3Avma)%20new%3Ararity) got downshifted for Masters Editions.

Hell, Tron was downshifted to the Basic Land slot.

Reply:
Ok - so I'm happy to merge whatever. Can you discuss in the rules panel and let me know what the decision is!